docs/developer-guide/syntax-reference.rst

```diff
@@ -1241,6 +1241,18 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | CONCAT                 |  ``CONCAT(col1, '_hello')``                                               | Concatenate two strings.                          |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| CURRENTDATE            |  ``CURRENTDATE()``                                                        | Gets an integer representing days since epoch.    |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| UNIX_TIMESTAMP         |  ``UNIX_TIMESTAMP([TIMEZONE])``                                           | Gets the Unix timestamp in milliseconds,          |
+|                        |                                                                           | represented as a BIGINT.                          |
+|                        |                                                                           | TIMEZONE is an optional parameter and it is a     |
+|                        |                                                                           | ``java.util.TimeZone`` ID format, for example:    |
+|                        |                                                                           | "UTC", "America/Los_Angeles", "PDT",              |
+|                        |                                                                           | "Europe/London".                                  |
+|                        |                                                                           | Notice that the returned timestamp might be       |
```

Comment:
Suggest: "The returned timestamp might be..."

Reply:
Done